Marble Blast Platinum/platinum/server/scripts/modes/gemMadness.cs

```diff
@@ -86,8 +90,18 @@
 function Mode_GemMadness::shouldRestartOnOOB(%this, %object) {
 	// If singleplayer, gem madness restarts. In multiplayer
 	// you wouldn't restart the whole mission if someone screws up.
-	return ($Server::ServerType $= "SinglePlayer");
+	//return ($Server::ServerType $= "SinglePlayer");
```

Comment:
You can just delete the old comments and put something like "never restart", don't have to leave the old stuff in there